samples/dashboard/demos/dashboard-layout/demo.html

```diff
@@ -1,7 +1,5 @@
-<link rel="stylesheet" type="text/css" href="https://code.highcharts.com/css/dashboard/gui.css">
-
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
-<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
```

Comment:
If this is a bootstrap demo the folder name should change. Dashboard itself should work without third-party libraries.

Reply:
Created ticket in Asana